docs/quickstart.md

```diff
@@ -0,0 +1,228 @@
+        {
+            "strain_id": "strain_id_1", # (1)!
+            "strain_alias": ["bgc_id_1", "spectrum_id_1", ...] # (2)!
+        },
+        {
+            "strain_id": "strain_id_2",
+            "strain_alias": ["bgc_id_2", "spectrum_id_2", ...]
+        },
+        ...
+    ],
+    "version": "1.0" # (3)!
+}
+```
+
+1. `strain_id` is the unique identifier of the strain.
+2. `strain_alias` is a list of aliases of the strain, which are the identifiers of the BGCs and
+   spectra of the strain.
+3. `version` is the schema version of this file. It is recommended to use the latest version of the
+   schema. The current latest version is `1.0`. 
+
+The BGC id is same as the name of the BGC file in the `antismash` directory, for example, given a 
+BGC file `xxxx.region001.gbk`, the BGC id is `xxxx.region001`.
+
+The spectrum id is same as the scan number in the `spectra.mgf` file in the `gnps` directory, 
+for example, given a spectrum in the mgf file with a scan `SCANS=1`, the spectrum id is `1`. 
+
+If you labelled the mzXML files (input for GNPS) with the strain id, you may need the function [extract_mappings_ms_filename_spectrum_id][nplinker.metabolomics.utils.extract_mappings_ms_filename_spectrum_id] 
+to extract the mappings from mzXML files to the spectrum ids.
+
+For the `local` mode, you need to create this file manually and put it in the working directory.
+It takes some effort to prepare this file manually, especially when you have a large number of strains.
+
+
+## 4. Prepare config file
+
+The configuration file `nplinker.toml` is required by NPLinker to specify the working directory, mode,
+and other settings for the run of NPLinker.
+
+
+The details of all settings can be found at this page [Config File](./concepts/config_file.md).
+
+To keep it simple, [default settings](./concepts/config_file.md#default-configurations) will be used automatically by NPLinker if you don't set them in your `nplinker.toml` config file.
+
+What you need to do is to set the `root_dir` and `mode` in the `nplinker.toml` file.
+
+=== "`local` mode"
+    ```toml title="nplinker.toml"
+    root_dir = "absolute/path/to/working/directory" # (1)!
+    mode = "local"
+    # and other settings you want to override the default settings 
+    ```
+
+    1. Replace `absolute/path/to/working/directory` with the **absolute** path to the working directory
+       created in step 2.
+
+
+=== "`podp` mode"
+    ```toml title="nplinker.toml"
+    root_dir = "absolute/path/to/working/directory" # (1)!
+    mode = "podp"
+    podp_id = "podp_id" # (2)!
+    # and other settings you want to override the default settings 
+    ```
+
+    1. Replace `absolute/path/to/working/directory` with the **absolute** path to the working directory
+       created in step 2.
+    2. Replace `podp_id` with the identifier of the dataset in the Paired Omics Data Platform (PODP).
+
+When you have prepared the `nplinker.toml` file, put it in the working directory.
```

Comment:
I'd move this information at the beginning of `4. Prepare config file`, rephrasing the sentence with something like: "The configuration file `nplinker.toml` is required by NPLinker to specify the working directory, mode, and other settings for the run of NPLinker. Once prepared, the configuration file must be placed in the working directory created in step 2."

Reply:
Done